apps/coscrad-frontend/src/components/resources/songs/song-detail.full-view.presenter.tsx

```diff
@@ -21,27 +25,14 @@ export const SongDetailFullViewPresenter = ({
     );
 
     return (
-        <div data-testid={id}>
-            <Card className="detail-card">
-                <div id="detail-term" className="detail-meta">
-                    {formatBilingualText(title, titleEnglish)}
-                </div>
-                <Divider id="detail-divider" />
-
-                <div className="detail-meta">
-                    <h3 className="detail-headers">Contributions:</h3>
-                    TODO use config for this
-                </div>
-                <div className="detail-meta">
-                    {typeof lyrics === 'string' && <SongLyrics lyrics={lyrics} />}
-                </div>
-                <div id="media-player">
-                    <MediaPlayer listenMessage="Play" audioUrl={audioURL} />
-                </div>
-                <div className="detail-meta">
-                    <CreditsHack resourceId={id} creditsMap={creditsMap} />
-                </div>
-            </Card>
-        </div>
+        <ResourceDetailFullViewPresenter name={name} id={id} type={ResourceType.song}>
+            <SingleOptionalPropertyPresenter display="Lyrics" value={lyrics} />
```

Comment:
I think the SingleOptionalPropertyPresenter just displays the string if it's defined (not null or undefined). But we really want to have a custom presenter for the lyrics. I'd be careful given that we may be looking at several paragraphs here.